.github/workflows/main.yml

```diff
@@ -76,7 +77,7 @@ jobs:
             os: windows-2019
             buildType: RELEASE
             options: config/azure/options.windows
-            dependenciesURL: "https://github.com/hypothetical-inc/gafferDependencies/releases/download/0.54.1.0/gafferDependencies-0.54.1.0-windows-msvc2017.zip"
+            dependenciesURL: "https://github.com/hypothetical-inc/gafferDependencies/releases/download/2.1.0/gafferDependencies-2.1.1-Python2-windows.zip"
```

Comment:
This has a 2.2.1 zip within a 2.1.0 folder. Is that a type or expected?

Reply:
Good catch, that's a typo. I updated the build tag to be 2.1.1 and updated the Github workflow in 90adede9480e8cd44b22f9431443c55946a25a1f